DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -288,7 +343,17 @@ private void AddProgressRecordForGroupWithSharedCourse()
                 @"SET IDENTITY_INSERT dbo.Progress ON
                     INSERT INTO Progress(ProgressID, CandidateID, CustomisationID, CustomisationVersion, SubmittedTime, EnrollmentMethodID, SupervisorAdminID)
                     VALUES (285172,299228,25918,1,GETUTCDATE(),3,0)
-                    SET IDENTITY_INSERT dbo.Progress OFF");
+                    SET IDENTITY_INSERT dbo.Progress OFF"
+            );
+        }
+
+        private string? GetGroupDescriptionById(int groupId)
+        {
+            return connection.Query<string?>(
+                @"SELECT GroupLabel FROM Groups 

```

Comment:
This is the wrong column, we should be getting GroupDescription (and you'll find that this causes the "doesn't update" unit test to fail unless you revert the changes to the test databases caused by the incorrect tests